### DISCUSSION TITLE
Help with Forced Combinations

### DISCUSSION THREAD

**Discussion by serserfan** [nftchef/art-engine#203] [Help + Q&A]
I want to use forced combinations in my project and I made an adjustment as follows.

"Cyristalized Blue": ["Thorn Blue", "Silver Thorn", "xxxxxxxx"],

However, only "Silver Thorn" appears when "Cyristalized Blue" is selected. Other options are not coming at all. Am I doing wrong somewhere?

**Comment by BarryDB83** [comment]
Maybe the layer order? Both ‘Thorn Blue’ and ‘xxxxxxx’ have to come after ‘crystallized blue’ for this rule to work

**Comment by serserfan** [comment]
Thank you, I will work on it again

**Reply by BarryDB83** [reply]
You can also solve this problem with nested folders. For example, make ‘crystallized blue’ a folder instead of image and add weight to it. Inside that folder you put images for ‘crystallized blue’, ‘thorn blue’ and ‘xxxxxx’. Don’t add weight to the images.

This way, if the folder gets picked, it will use all images inside

**Comment by cryptofannatic** [comment]
@serserfan did you manage to figure this out?

**Reply by serserfan** [reply]
Hi @cryptofannatic , I solved my problem using a different library. If you're interested in jalagar/animated-art-engine/
